test/image/render_mocks_cli

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+
+programname=$0
+
+function usage {
+    echo "usage: $programname output_folder [docker:image]"
+    echo "  it renders all the JSON files found in ./test/image/mocks into the output_folder."
+    echo "  if docker:image is provided, rendering is done in the given Docker image."
+    exit 1
+}
+
+# if less than two arguments supplied, display usage
+if [  $# -le 0 ]
+then
+	usage
+fi
+
+for format in png pdf svg eps emf
+do
+  # To obtain reproducible results, we need to use software rendering
+  export LIBGL_ALWAYS_SOFTWARE=true
+  export GALLIUM_DRIVER=softpipe
+
+  # Supply defaults assets to Orca
+  ORCA_OPTS=(--topojson /plotly-geo-assets.js --mathjax /mathjax/MathJax.js)
+
+  # pin Plotly.js' version
+  ORCA_OPTS+=(--plotlyJS "https://cdn.plot.ly/plotly-1.47.2.min.js")
+
+  # Set Orca options
+  ORCA_OPTS+=(--verbose --format "$format" --output-dir "$1")
+
+  if [ -z "$2" ]; #
+  then
+    echo "Executing locally"
```

Comment:
Could you log info about the format tested?

The log currently look like:

![image](https://user-images.githubusercontent.com/6675409/56974374-baefd380-6b3c-11e9-9ccf-65f33a3ad209.png)

so maybe having _Executing locally. Format $format_ would be an improvement.

Reply:
![image](https://user-images.githubusercontent.com/6675409/56974511-00140580-6b3d-11e9-952f-e9703f38cc59.png)

I was expecting a slower job. Nicely done!

Reply:
Done in 3a37405b6932fd0d5659ebfdd6a1080ba4914580

Reply:
:tada: